sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/query.sql

```diff
@@ -278,6 +278,11 @@ WHERE
   submission_date = @submission_date
   -- Filter out events associated with suspiciously active clients.
   AND NOT (user_event_count > 50 AND event_type = 'click')
+  AND IF(
+    DATE_DIFF(CURRENT_DATE(), DATE('@submission_date'), DAY) > 30,
```

Comment:
Someone should check whether 30 days is the retention policy here

Reply:
It is: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/main/templates/contextual-services/defaults.schema.json#L4